local_deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+ 
+# Usually "pr-<PR number>" - e.g., "pr-1600" 
+SUFFIX="pr-1666"
```

Comment:
I think if we want to commit this script we should:
- parameterize and require the pr suffix parameter, so we prevent accidental prod deploys
- have code shared between this script and the actual production.yml job so things don't go out of sync if/when we make changes

I think it might be better to write a dev task ticket for this and add this in it's own PR

Reply:
sounds good. I'll remove the file from this PR and write a dev task ticket for it instead

Reply:
Ticket created here: https://github.com/bcgov/wps/issues/1670